index.html

```diff
@@ -50,16 +48,11 @@ <h2>Things that Betina dislikes</h2>
         <li><strong>Being dirty:</strong> Betina may not like getting dirty and may not enjoy playing in mud or dirty environments.</li>
         <li><strong>Strict discipline:</strong> Betina may not respond well to strict discipline or harsh training methods. She may be better trained with positive reinforcement and rewards-based methods.</li>
         </ol>
-
         <hr>
-
         <div>
         <button class="btn">Click if you liked<img src="imagenes/thumbs-up-solid.svg" alt="ThumbsUp" class="thumbs"></button>
         </div>
-
-
-
 </main>
 
```

Comment:
- [ ] Hi, nice one with the reviews, kindly note that from the requirement is that you use Github inline comments for reviews, Kindly, ensure you use inline GitHub comments for at least one of the reviews. This shows professionalism, you can check [this](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) video for how. 😀.